runtime/tests/staking_reward.rs

```diff
@@ -549,3 +550,50 @@ fn authorship_reward_of_a_chilled_validator() {
 			);
 		});
 }
+
+#[test]
+/// This tests if slash goes to treasury as CENNZ.
+fn slashed_cennz_gets_into_treasury() {
+	let validators = make_authority_keys(6);
+	let initial_balance = 1_000 * DOLLARS;
+
+	ExtBuilder::default()
+		.slash_defer_duration(0)
```

Comment:
Shouldn't this be 168 or even not specified to get its default value?

Reply:
The defer duration is set as constant - pub const SlashDeferDuration: crml_staking::EraIndex = 24 * 7 at https://github.com/cennznet/cennznet/blob/b40b1afbd7a226823e08fb069b3b0b3ba92592aa/runtime/src/lib.rs#L313 so can't change it... I have reverted all the code which was used to set it to configurable value...